src/main/java/core/basesyntax/Random.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Random {
```

Comment:
we don't need this class. Color supplier is responsible for generating random color, it will use standard randomizer. Lottery will use it one too